src/SOS/SOS.Extensions/Clrma/ExceptionWrapper.cs

```diff
@@ -0,0 +1,211 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Runtime.Utilities;
+
+namespace SOS.Extensions.Clrma
+{
+    public sealed class ExceptionWrapper : COMCallableIUnknown
```

Comment:
Do we need to put a managed<->native marshalling layer into SOS for this? My understanding was that CLRMA was able to interact with CLRMD directly using its managed API - couldn't we do the same when it interacts with SOS?

Reply:
The CLRMA interfaces are all C++/COM.  There was an managed C++ interop wrapper dll and C# provider that used CLRMD but it was abandoned and removed.  It seemed a lot simpler to just do the interop myself since SOS already does this a lot.